.github/workflows/ci.yml

```diff
@@ -6,21 +6,20 @@ jobs:
     build-mlibc:
         strategy:
             matrix:
-                arch: [x86_64, riscv64, aarch64]
+                arch: [x86_64, aarch64]
```

Comment:
Why is ``riscv`` removed?

Reply:
the port isn't complete enough to configure a cross compiler

Reply:
unsure, but this is the build log:
```
239.5 In file included from ../../../tool-src/libgcc/unwind-dw2.c:410:
239.5 ./md-unwind-support.h: In function 'riscv_fallback_frame_state':
239.5 ./md-unwind-support.h:67:6: warning: assignment to 'struct sigcontext *' from incompatible pointer type 'mcontext_t *' [-Wincompatible-pointer-types]
239.5    67 |   sc = &rt_->uc.uc_mcontext;
239.5       |      ^
239.5 ./md-unwind-support.h:77:53: error: invalid use of undefined type 'struct sigcontext'
239.5    77 |       fs->regs.reg[i].loc.offset = (_Unwind_Ptr) &sc->gregs[i] - new_cfa;
239.5       |                                                     ^~
239.5 ./md-unwind-support.h:84:21: error: invalid use of undefined type 'struct sigcontext'
239.5    84 |     (_Unwind_Ptr) sc->gregs[0] - new_cfa;
239.5       |                     ^~
239.5 make[2]: *** [../../../tool-src/libgcc/static-object.mk:17: unwind-dw2.o] Error 1
```

... slightly mangled by docker.  to reproduce it, you can use the script in mlibc-crossers by just adding a riscv target

Reply:
That seems lie an easy fix, seems like we're just missing a ``struct sigcontext``.

Reply:
sure, but I have no knowledge of riscv64, nor a way to test it.

Reply:
Right, but I'm also not going to merge a regression on the CI. I'll submit a PR to your branch later to fix this.

Reply:
This should get fixed once #945 is merged.